engine/src/io/data_parser/sql/sqlcommon.h

```diff
@@ -0,0 +1,42 @@
+#include <cudf/column/column_factories.hpp>
+#include <cudf/detail/utilities/vector_factories.hpp>
+#include <cudf/io/types.hpp>
+#include <cudf/strings/convert/convert_datetime.hpp>
+#include <cudf/utilities/bit.hpp>
+
+namespace ral {
+namespace io {
+
+struct cudf_string_col {
+  std::vector<char> chars;
+  std::vector<cudf::size_type> offsets;
+};
+
+
+static inline std::unique_ptr<cudf::column>
+build_str_cudf_col(cudf_string_col *host_col,
+                   const std::vector<cudf::bitmask_type> &null_mask) {
+  auto d_chars = cudf::detail::make_device_uvector_sync(host_col->chars);
+  auto d_offsets = cudf::detail::make_device_uvector_sync(host_col->offsets);
+  auto d_bitmask = cudf::detail::make_device_uvector_sync(null_mask);
+  return cudf::make_strings_column(d_chars, d_offsets, d_bitmask);
+}
+
+template <typename T>
+static inline std::unique_ptr<cudf::column>
+build_fixed_width_cudf_col(size_t total_rows,
```

Comment:
i think there are already cudf helpers for this and this doesnt seem to be related to sql

Reply:
we could not found the builder or a sample code, perhaps we can move this to CudfUtils somewhere ... do you know where I can found this cudf helpers (there must not be the ones in cudf::testing) cc @rommelDB